docs/api-reference.md

```diff
@@ -906,3 +906,259 @@ Fixes URLs passed to the specified props to correct the static resource folder p
+
+```js
+import { getClient } from 'resolve-client'
+
+const main = async resolveContext => {
+  const client = getClient(resolveContext)
+  ...
+```
+
+The `getClient` function takes a reSolve context as a parameter and returns an initialized client object. This object exposes the following functions:
+
+| Function Name                           | Description                                |
+| --------------------------------------- | ------------------------------------------ |
+| [command](#command)                     | Sends an aggregate command to the backend. |
+| [query](#query)                         | Queries a Read Model.                      |
+| [getStaticAssetUrl](#getstaticasseturl) | Gets a static file's full URL.             |
+| [subscribe](#subscribe)                 | Subscribes to View Model updates.          |
+| [unsubscribe](#unsubscribe)             | Unsubscribes from View Model updates.      |
+
+#### command
+
+Sends an aggregate command to the backend.
+
+##### Example
+
+```js
+client.command(
+  {
+    aggregateName: 'Chat',
+    type: 'postMessage',
+    aggregateId: userName,
+    payload: message
+  },
+  err => {
+    if (err) {
+      console.warn(`Error while sending command: ${err}`)
+    }
+  }
+)
+```
+
+#### query
+
+Queries a Read Model.
+
+##### Example
+
+```js
+const { data } = await client.query({
+  name: 'chat',
+  aggregateIds: '*'
+})
+```
+
+#### getStaticAssetUrl
+
+Gets a static file's full URL.
+
+##### Example
+
+```js
+var imagePath = client.getStaticAssetUrl('/account/image.jpg')
+```
+
+#### subscribe
+
+Subscribes to View Model updates. Returns a promise that resolves to a **subscription** object.
+
+##### Example
+
+```js
+const chatViewModelUpdater = event => {
+  const eventType = event != null && event.type != null ? event.type : null
+  const eventHandler = chatViewModel.projection[eventType]
+
+  if (typeof eventHandler === 'function') {
+    chatViewModelState = eventHandler(chatViewModelState, event)
+  }
+
+  setImmediate(updateUI.bind(null, chatViewModelState))
+}
+
+await client.subscribe('chat', '*', chatViewModelUpdater)
+```
+
+#### unsubscribe
+
+Unsubscribes from View Model updates.
+
+##### Example
+
+```js
+await client.unsubscribe(subscription)
+```
+
+### resolve-react-hooks library
+
+The **resolve-react-hooks** library provides React hooks that you can use to connect React components to a reSolve backend. The following hooks are provided.
+
+| Hook                                    | Description                                                              |
+| --------------------------------------- | ------------------------------------------------------------------------ |
+| [useCommand](#useCommand)               | Initializes a command that can be passed to the backend.                 |
+| [useCommandBuilder](#useCommandBuilder) | Allows to generate commands based on input parameters.                   |
```

Comment:
Who does it allow to generate commands? Is it "you" (the reader/developer)?

Reply:
Changed to "Allows a component to"